src/Filament/Traits/HasExcerpt.php

```diff
@@ -18,7 +18,15 @@ public function excerpt(): Attribute
         $paragraph = '';
         foreach ($content as $key => $value) {
             if ($value['type'] === 'paragraph') {
```

Comment:
Suggest using the TipTap helper here:  `tiptap_converter()->asText($value['content'])`